mlos_bench/mlos_bench/storage/storage_factory.py

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""
+Factory method to create a new Storage instance from configs.
+"""
+
+from typing import Any, Optional, List, Dict
+
+from mlos_bench.config.schemas import ConfigSchema
+from mlos_bench.services.config_persistence import ConfigPersistenceService
+from mlos_bench.tunables.tunable_groups import TunableGroups
+from mlos_bench.storage.base_storage import Storage
+
+
+def from_config(config_file: str,
+                global_configs: Optional[List[str]] = None,
+                tunables: Optional[List[str]] = None,
+                **kwargs: Any) -> Storage:
```

Comment:
As mentioned elsewhere, this requires the user to specify two things that are probably already in the `--config` file they used to launch `mlos_bench` CLI.
In the future at least it'd be nice to be able to also instantiate a Storage reader from just the `--config` file (assuming it already references the `globals_storage_auth.jsonc` file).

Reply:
yeah, I really wanted to do exactly that, but there are too many different components involved. Let's keep it as is for the demo, and we can always come back to it later.